coreos-base/coreos/coreos-0.0.1.ebuild

```diff
@@ -100,6 +86,7 @@ RDEPEND="${RDEPEND}
 	)"
 
 RDEPEND="${RDEPEND}
+	amd64? ( app-admin/etcd-wrapper )
```

Comment:
Ah, I branched before #2520 was merged. I've updated it to unconditionally include etcd-wrapper.